src/rust/engine/src/cffi_build.rs

```diff
@@ -76,7 +98,7 @@ fn mark_for_change_detection(path: PathBuf) -> PathBuf {
   path
 }
 
-fn make_flags(env_script_path: &Path) -> Result<Vec<String>> {
+fn make_flags(env_script_path: PathBuf) -> Result<Vec<String>> {
```

Comment:
This seems a strange change; this function doesn't appear to need ownership? (Not objecting, just curious :))

Reply:
Agreed, reverted.